packages/gitbook/src/components/DocumentView/Tabs/DynamicTabs.tsx

```diff
@@ -145,10 +159,44 @@ export function DynamicTabs(props: {
                 return;
             }
 
-            selectTab(tabPanel.id);
+            selectTab(tabPanel.id, false);
+        } else {
+            const firstTab = tabs[0];
+            if (firstTab) {
+                selectTab(firstTab.id, false);
```

Comment:
What's this for, why check if firstTab is defined? Is it so that you confirm that the tabs actually exist in the DOM before you try and open the actual one that is opened?

Reply:
Yes it's useless.